cdc/sink/codec/maxwell.go

```diff
@@ -132,19 +196,14 @@ func (d *MaxwellEventBatchEncoder) AppendRowChangedEvent(e *model.RowChangedEven
 	if err != nil {
 		return EncoderNoOperation, errors.Trace(err)
 	}
-
 	var keyLenByte [8]byte
 	binary.BigEndian.PutUint64(keyLenByte[:], uint64(len(key)))
 	var valueLenByte [8]byte
 	binary.BigEndian.PutUint64(valueLenByte[:], uint64(len(value)))
 
-	d.keyBuf.Write(keyLenByte[:])
-	d.keyBuf.Write(key)
-
-	d.valueBuf.Write(valueLenByte[:])
 	d.valueBuf.Write(value)
 
-	d.batchSize += 1
+	d.batchSize++
 	return EncoderNoOperation, nil
```

Comment:
@liuzix the batching is disable in maxwell, what operation should we return?

Reply:
`EncoderNeedAsyncWrite` will instruct the mqSink to call `Build` immediately. So that is what we need.

Reply:
done